.docker/qgis3-qt5-build-deps.dockerfile

```diff
@@ -171,6 +170,26 @@ RUN apt-get update
 RUN ACCEPT_EULA=Y apt-get install -y --allow-unauthenticated msodbcsql18 mssql-tools18
 ENV PATH="/opt/mssql-tools18/bin:${PATH}"
 
+# PDAL is not available in ubuntu 24.04
+# Install it from source
+# PDAL dependencies
+RUN  apt-get update \
+     && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     ninja-build \
+     libgdal-dev \
+     libproj-dev
+# download PDAL and compile it
+ENV PDAL_VERSION=2.8.4
```

Comment:
I would switch to `ARG` since it's more a build time variable. And I would move it at the top of the dockerfile to increase readability.

See:

- https://docs.docker.com/build/building/variables/
- https://www.docker.com/blog/docker-best-practices-using-arg-and-env-in-your-dockerfiles/
- https://vsupalov.com/docker-arg-env-variable-guide/

Reply:
Done.